tests/phpunit/Civi/Core/SettingsManagerTest.php

```diff
@@ -143,4 +143,17 @@ protected function createManager() {
     return $manager;
   }
 
+  /**
+   * Test passing in an array to Civi::settings facade
+   */
+  public function testArraySetting() {
+    $domain = \CRM_Core_DAO::createTestObject('CRM_Core_DAO_Domain');
+
+    $manager = $this->createManager()->useDefaults();
+    $dSettings = $manager->getBagByDomain($domain->id);
+    $dSettings->set('test_setting', ['hello' => 'World']);
```

Comment:
I merged this but since there is not metadata for test_setting it should have failed with a deprecation notice IMHO